src/main/java/com/microsoft/azure/vmagent/AzureVMManagementServiceDelegate.java

```diff
@@ -2512,7 +2512,7 @@ public static CloudStorageAccount getCloudStorageAccount(StorageAccount storageA
         try {
             return new CloudStorageAccount(
                     new StorageCredentialsAccountAndKey(storageAccount.name(), storageAccountKey),
-                    false,
+                    true,
```

Comment:
should it be an option to avoid any change on existing users?

Reply:
This method is transparent to user. And I have confirmed with SDK teams that all storage account support https communication, so this change won't block existing users.